Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  	<title>GirlDevelopIt</title>
+    <meta charset="utf-8"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+	<!-- jQuery library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<!-- Popper JS -->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"></script>
+	<!-- Latest compiled JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+	<script src="js/script.js"></script>
+	<link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+  	<div class="container">
+  	<a href="https://www.girldevelopit.com/">
+  		<img src="https://stackoverflow.blog/wp-content/uploads/2017/08/wordmark-tagline-pink-2.png" alt="girldevelopit" style="width:30%"><br><br>
+  	</a>
+  	<h1>About</h1>
+  		<p>It can be intimidating for women to learn and ask questions when they are in an <b>extreme minority</b>. While open and welcoming, today's budding developer community is up to <b>91% male</b>. If we empower more females with the confidence in their technological capabilities <i>we can begin to change this landscape</i>.</p>
+  	<h2>Our Locations</h2>
+  		<ul style="list-style-type:disc">
```

Comment:
You should not have any inline styles. Besides, you already have a CSS file, so just move all your inline styles to that file.